pom.xml

```diff
@@ -1,16 +1,40 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
+
 	<groupId>org.reactome.datapipeline</groupId>
 	<artifactId>orthoinference</artifactId>
 	<version>1.0.2-SNAPSHOT</version>
-	<name>Orthoinference</name>
+	<packaging>jar</packaging>
+
+	<name>orthoinference</name>
+	<description>Orthology projection of human reactions and pathways to non-human species</description>
```

Comment:
I think we should include that it is based on PANTHER in the description

Reply:
Sure, I can add it in.

Reply:
Done: https://github.com/reactome/release-orthoinference/pull/1/commits/86aaedd082034456644f459b77170bf560924194